docs/resources/elasticsearch_script.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Cluster"
+layout: ""
+page_title: "Elasticstack: elasticstack_elasticsearch_script Resource"
+description: |-
+  Creates or updates a stored script or search template.
+---
+
+# Resource: elasticstack_elasticsearch_script
+
+Creates or updates a stored script or search template. See https://www.elastic.co/guide/en/elasticsearch/reference/current/create-stored-script-api.html
+
+## Example Usage
```

Comment:
Should have said this earlier sorry, having a search template example would be nice too.